src/schemas/filtersContainer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "target": {
+      "$ref": "#target"
+    },
+    "filters": {
+      "type": "array",
+      "items": {
+        "$ref": "#filter"
+      }
+    }
+  },
+  "required": [
+    "filters"
```

Comment:
Since filters are the only required property and target is not required, does it default to "report" by default?